modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/succubus_milk.dm

```diff
@@ -99,93 +99,140 @@
 		if(mob_breasts.visibility_preference == GENITAL_ALWAYS_SHOW || exposed_mob.is_topless())
 			switch(translation)
 				if(BREAST_SIZE_FLATCHESTED)
-					return
 				if(BREAST_SIZE_BEYOND_MEASUREMENT)
 					exposed_mob.visible_message(span_notice("[exposed_mob]'s [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(public_bigger_action_text_list)]"))
 					to_chat(exposed_mob, span_purple("Your [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(action_text_list)]about [mob_breasts.genital_size] inches in diameter."))
-					return
 				else
 					if(mob_breasts?.genital_size >= (max_breast_size - 2))
 						exposed_mob.visible_message(span_notice("[exposed_mob]'s [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(public_bigger_action_text_list)]"))
 						to_chat(exposed_mob, span_purple("Your [pick(words_for_bigger)] [pick(bigger_boob_text_list)] [pick(action_text_list)]about [translation]-cups."))
-						return
 					else
 						exposed_mob.visible_message(span_notice("[exposed_mob]'s [pick(boob_text_list)] [pick(public_action_text_list)]"))
 						to_chat(exposed_mob, span_purple("Your [pick(boob_text_list)] [pick(action_text_list)]about [translation]-cups."))
-						return
 		else
 			switch(translation)
 				if(BREAST_SIZE_FLATCHESTED)
-					return
```

Comment:
I feel like all of these switch statements should be moved into a separate proc called by this one, so you can preserve all of the return statements. Right now this looks very messy, and there's far too much indentation everywhere in there.

Reply:
Yeah, fair enough. I just want to point out that I did not write this but I shall atone for their sins.